apps/vr-tests-web-components/src/stories/menu-list/menu-list.stories.tsx

```diff
@@ -0,0 +1,194 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28864&commitId=fa1d062b0ffed211d992f4e23e4c4ec6735560f7&env=prod&clientType=FLUENTUI)


  <details><summary><b>MenuList</b> 47 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox - Dark Mode.1st selected.chromium.png|0| Added|
|MenuList.Checkbox - Dark Mode.2nd selected.chromium.png|0| Added|
|MenuList.Checkbox - Dark Mode.normal.chromium.png|0| Added|
|MenuList.Checkbox - RTL.1st selected.chromium.png|0| Added|
|MenuList.Checkbox - RTL.2nd selected.chromium.png|0| Added|
|MenuList.Checkbox - RTL.normal.chromium.png|0| Added|
|MenuList.Checkbox With Icons - RTL.1st selected.chromium.png|0| Added|
|MenuList.Checkbox With Icons - RTL.2nd selected.chromium.png|0| Added|
|MenuList.Checkbox With Icons - RTL.normal.chromium.png|0| Added|
|MenuList.Checkbox With Icons.1st selected.chromium.png|0| Added|
|MenuList.Checkbox With Icons.2nd selected.chromium.png|0| Added|
|MenuList.Checkbox With Icons.normal.chromium.png|0| Added|
|MenuList.Checkbox.1st selected.chromium.png|0| Added|
|MenuList.Checkbox.2nd selected.chromium.png|0| Added|
|MenuList.Checkbox.normal.chromium.png|0| Added|
|MenuList.Default - Dark Mode.hover menuitem.chromium.png|0| Added|
|MenuList.Default - Dark Mode.normal.chromium.png|0| Added|
|MenuList.Default - RTL.hover menuitem.chromium.png|0| Added|
|MenuList.Default - RTL.normal.chromium.png|0| Added|
|MenuList.Default.hover menuitem.chromium.png|0| Added|
|MenuList.Default.normal.chromium.png|0| Added|
|MenuList.Radio - Dark Mode.1st selected.chromium.png|0| Added|
|MenuList.Radio - Dark Mode.2nd selected.chromium.png|0| Added|
|MenuList.Radio - Dark Mode.normal.chromium.png|0| Added|
|MenuList.Radio - RTL.1st selected.chromium.png|0| Added|
|MenuList.Radio - RTL.2nd selected.chromium.png|0| Added|
|MenuList.Radio - RTL.normal.chromium.png|0| Added|
|MenuList.Radio With Icons - RTL.1st selected.chromium.png|0| Added|
|MenuList.Radio With Icons - RTL.2nd selected.chromium.png|0| Added|
|MenuList.Radio With Icons - RTL.normal.chromium.png|0| Added|
|MenuList.Radio With Icons.1st selected.chromium.png|0| Added|
|MenuList.Radio With Icons.2nd selected.chromium.png|0| Added|
|MenuList.Radio With Icons.normal.chromium.png|0| Added|
|MenuList.Radio.1st selected.chromium.png|0| Added|
|MenuList.Radio.2nd selected.chromium.png|0| Added|
|MenuList.Radio.normal.chromium.png|0| Added|
|MenuList.With Icons.hover menuitem.chromium.png|0| Added|
|MenuList.With Icons.normal.chromium.png|0| Added|
|MenuList.With Submenu - Dark Mode.1st selected.chromium.png|0| Added|
|MenuList.With Submenu - Dark Mode.2nd selected.chromium.png|0| Added|
|MenuList.With Submenu - Dark Mode.normal.chromium.png|0| Added|
|MenuList.With Submenu - RTL.1st selected.chromium.png|0| Added|
|MenuList.With Submenu - RTL.2nd selected.chromium.png|0| Added|
|MenuList.With Submenu - RTL.normal.chromium.png|0| Added|
|MenuList.With Submenu.1st selected.chromium.png|0| Added|
|MenuList.With Submenu.2nd selected.chromium.png|0| Added|
|MenuList.With Submenu.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>